api/app/src/main/kotlin/packit/controllers/PacketController.kt

```diff
@@ -77,4 +81,29 @@ class PacketController(
             .headers(response.second)
             .body(response.first)
     }
+
+    @GetMapping("/{name}/{id}/zip")
```

Comment:
Do we really need the name here? Can't it be derived from the `id`?

Also I think it would be best to have the format as `/zip/{id}` to avoid name collisions (and change `getPacketsByName`).

Reply:
I’ve thought some more about it and maybe `/{id}/zip` is alright actually and consistent with the `file` and `metadata` one, as long as we shuffle the overlapping routes around eventually.

Reply:
I've started re-arranging these routes to make a bit more sense in https://github.com/mrc-ide/packit/pull/173

Reply:
Looks like we're moving to `/packets/<id>/file` for file downloads, and so I'll go for `/packets/<id>/zip`.